src/librustc/session/mod.rs

```diff
@@ -1317,7 +1317,7 @@ fn validate_commandline_args_with_session_available(sess: &Session) {
        sess.opts.prints.iter().all(|&p| p == PrintRequest::NativeStaticLibs) {
         sess.err("Profile-guided optimization does not yet work in conjunction \
```

Comment:
```suggestion
        sess.err("profile-guided optimization does not yet work in conjunction \
```
Let's make this quick drive-by fix while we're at it.